src/api/app/controllers/wizard_controller.rb

```diff
@@ -56,7 +56,7 @@ def package_wizard
     xml = create_service_file
 
     Suse::Backend.put("/source/#{params[:project]}/#{params[:package]}/_service?rev=upload&user=#{User.current.login}", xml)
-    Suse::Backend.post("/source/#{params[:project]}/#{params[:package]}?cmd=commit&rev=upload&user=#{User.current.login}", '')
+    Suse::Backend.post("/source/#{params[:project]}/#{params[:package]}?cmd=commit&rev=upload&user=#{User.current.login}")
```

Comment:
On Donnerstag, 22. September 2016, 05:55:30 CEST wrote Ana Mar�a Mart�nez G�mez:
> > @@ -56,7 +56,7 @@ def package_wizard
> >      xml = create_service_file
> >  
> >      Suse::Backend.put("/source/#{params[:project]}/#{params[:package]}/_service?rev=upload&user=#{User.current.login}", xml)
> > -    Suse::Backend.post("/source/#{params[:project]}/#{params[:package]}?cmd=commit&rev=upload&user=#{User.current.login}", '')
> > +    Suse::Backend.post("/source/#{params[:project]}/#{params[:package]}?cmd=commit&rev=upload&user=#{User.current.login}")
> 
> @adrianschroeter why is not the same?

'' will create a body, even though it will be empty.
It also has some effects on how we do logging backend 
requests. Have a look in write_backend_log() in 

 src/api/lib/opensuse/backend.rb

if you are interessted in that.

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany